src/features/projects/components/maps/Credits.tsx

```diff
@@ -19,18 +19,24 @@ export default function Credits({ setCurrencyCode }: Props): ReactElement {
     const [selectedCurrency, setSelectedCurrency] = React.useState('EUR');
     const [selectedCountry, setSelectedCountry] = React.useState('DE');
     const [openLanguageModal, setLanguageModalOpen] = React.useState(false);
+    
 
     const handleLanguageModalClose = () => {
         setLanguageModalOpen(false);
     };
     const handleLanguageModalOpen = () => {
         setLanguageModalOpen(true);
     };
+    const router = useRouter();
+    const embedVal = router.query.embed === 'true';
```

Comment:
@prachigarg19 Rename `embedVal` to `isEmbed` as it contains a boolean value that indicates if the page is embedded.